.vscode/tasks.json

```diff
@@ -10,7 +10,10 @@
         "kind": "build",
         "isDefault": true
       },
-      "command": "node common/scripts/install-run-rush.js build --verbose",
+      "command": "npx gulp buildWithoutPackage --verbose",
```

Comment:
I dont think we need `npx` here. We can assume users have already run `npm install` before running in vs code. So, just do the same, but remove `npx`. You may need to add a full path here, so `${workspaceRoot}/extensions/ql-vscode/node_modules/.bin/gulp`.

Reply:
I sort of like `npx` as a terse idiom here, but I don't mind being more explicit if you think it's better to avoid the magic of `npx` auto-downloading packages. I'm pretty sure we already need to be in the correct `cwd` to have gulp work, so a path relative to `extensions/ql-vscode` should work.

Reply:
My concern with npx is with versioning. I've had situations before where people ran into different behaviour because `npx` had installed different versions. If we are running from the `node_modules` directory, this won't happen.